deel-2-flexbox-layout/opdracht-2/styles/styles.css

```diff
@@ -1 +1,40 @@
-/*Schrijf hier jouw CSS*/
\ No newline at end of file
+body{
+    display: flex;
+    flex-direction: row;
+    min-height: 100vh;
+}
+
+div{
```

Comment:
In dit geval was div mogelijk maar eigenlijk wilde je de class col-2 gebruiken zodat het ook alleen op die div betrekking had.